src/gpm-tray-icon.c

```diff
@@ -391,7 +392,9 @@ gpm_tray_icon_create_menu (GpmTrayIcon *icon)
 	gtk_style_context_add_class(context,"mate-panel-menu-bar");
 
 	/* about */
-	item = gtk_image_menu_item_new_from_stock ("gtk-about", NULL);
+	item = mate_image_menu_item_new_with_mnemonic (_("_Info"));
```

Comment:
Using ```_About```  is more reasonable

Reply:
I was surprised for myself. The applet used `Info` before, see 1.26 branch. For that reason i used the old name.
But i can change it. I don't mind.

Reply:
@raveit65  made an error submission in the master branch. Can you help me delete this submission
https://github.com/mate-desktop/mate-power-manager/commit/f9a53963f410c8967c0ff29f2d6dea7fff1a3356

Reply:
I reverted the commit.
Deleting a commit from master is ugly, because this can destroy other peoples git history if they got made a pull on master.

Reply:
Thank you

Reply:
I changed label to `_About`